api/v1/openapi.yaml

```diff
@@ -2533,6 +2533,9 @@ definitions:
       num-shared-services:
         description: Number of services in the cluster
         type: integer
+      num-service-exports:
+        description: Number of MCS-API service exports in the cluster
+        type: integer
```

Comment:
It could make sense to additionally propagate the `ServiceExportsEnabled` capability in the appropriate section (especially if set to false when the feature is disabled).